css/index.css

```diff
@@ -3,28 +3,35 @@
 }
 
 .whole {
-    padding: 10px 3%;
-    margin: auto; 
+    padding: 10px;
+    margin: 10px;
 }
+/* .about {
+    margin-top: 50px;
+} */
 
 header {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding:10px 20px;
+    /* padding:10px; */
     position: fixed; 
     top: 0;
+    max-width: 500% ;
```

Comment:
Just curious, why did you end up needing to set the `max-width` of this component so high? Since it has `width: 100%` it's already stretched go across the whole width of the screen (effectively)